internal/build/phase_config_provider.go

```diff
@@ -65,6 +77,10 @@ func NewPhaseConfigProvider(name string, lifecycleExec *LifecycleExecution, ops
 
 	provider.ctrConf.Cmd = append([]string{"/cnb/lifecycle/" + name}, provider.ctrConf.Cmd...)
 
+	// `–security-opt=”apparmor=unconfined” –cap-add=SYS_PTRACE`
+	provider.hostConf.CapAdd = strslice.StrSlice{"SYS_PTRACE"}
+	provider.hostConf.SecurityOpt = []string{"apparmor=unconfined"}
```

Comment:
Should we only set this if there are any `Ports` exposed? I assume leaving `SYS_PTRACE` may open us up to a vuln of some sort, though setting `apparmor` may be good for our security posture, but I admittedly don't know much about either